cmd/orb.go

```diff
@@ -1410,10 +1431,12 @@ func initOrb(opts orbOptions) error {
 	}
 
 	// Push a dev version of the orb.
-	_, err = api.CreateOrb(opts.cl, namespace, orbName, opts.private)
+	if !orbExists {
+		_, err = api.CreateOrb(opts.cl, namespace, orbName, opts.private)
 	if err != nil {
 		return errors.Wrap(err, "Unable to create orb")
 	}
```

Comment:
I believe the indentation is wrong here.

Reply:
You'd think go fmt would say something here but it doesnt.
Im not sure if it's a GitHub.com thing.. it doesnt appear this way to me locally?
<img width="688" alt="image" src="https://user-images.githubusercontent.com/33272306/196799113-6d4c4f25-b7e4-4224-a942-26050e895fb0.png">

Reply:
Cloned the PR, and still seeing the unformatted lines 🤔 

![image](https://user-images.githubusercontent.com/21243061/196812955-a2f35d94-fc18-4972-9f99-5a504d5dfb03.png)

Reply:
Is GitHub, perhaps, not considering the file to have updated because of a change in whitespace/indentation, @KyleTryon?

Reply:
downloading and running `golangci-lint run cmd/orb.go` has given me a few hits on the file but completely unrelated to this error, which interestingly is also the _same_ tool that failed in CI with the same issue you all are mentioning. Im not sure why my local version refuses to show this. going to wipe my local version and re-clone